core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -2617,11 +2617,24 @@ private void parseIndexing(final XMLExtendedStreamReader reader, final Configura
          String value = reader.getAttributeValue(i);
          Attribute attribute = Attribute.forName(reader.getAttributeLocalName(i));
          switch (attribute) {
+            case ENABLED:
+               if (reader.getSchema().since(11, 0)) {
+                  builder.indexing().enabled(Boolean.parseBoolean(value));
+               } else {
+                  throw ParseUtils.unexpectedAttribute(reader, i);
+               }
+               break;
             case INDEX:
+               if (reader.getSchema().since(11, 0)) {
+                  CONFIG.indexModeDeprecated();
+               }
                if ("LOCAL".equals(value)) {
                   value = "PRIMARY_OWNER";
                   CONFIG.indexLocalIsNotSupported();
                }
+               if ("NONE".equals(value)) {
```

Comment:
These 3 lines are not needed because builder.indexing().index(index) achieves the same thing

Reply:
removed